cmd/libs/go2idl/client-gen/generators/fake/generator_fake_for_type.go

```diff
@@ -148,49 +152,85 @@ func (g *genFakeForType) GenerateType(c *generator.Context, t *types.Type, w io.
 		"ExtractFromListOptions":         c.Universe.Function(types.Name{Package: pkgClientGoTesting, Name: "ExtractFromListOptions"}),
 	}
 
-	noMethods := extractBoolTagOrDie("noMethods", t.SecondClosestCommentLines) == true
-
-	readonly := extractBoolTagOrDie("readonly", t.SecondClosestCommentLines) == true
+	if len(tags.OnlyVerbs) > 0 {
+		tags.SkipVerbs = []string{}
+		for _, m := range util.SupportedVerbs {
+			skip := true
+			for _, o := range tags.OnlyVerbs {
+				if o == m {
+					skip = false
+					break
+				}
+			}
+			if skip {
+				tags.SkipVerbs = append(tags.SkipVerbs, m)
+			}
+		}
+	}
 
-	if namespaced {
-		sw.Do(structNamespaced, m)
-	} else {
+	if tags.NonNamespaced {
 		sw.Do(structNonNamespaced, m)
+	} else {
+		sw.Do(structNamespaced, m)
 	}
 
-	if !noMethods {
+	if !tags.NoVerbs {
 		sw.Do(resource, m)
 		sw.Do(kind, m)
 	}
 
-	if !noMethods && !readonly {
-		sw.Do(createTemplate, m)
-		sw.Do(updateTemplate, m)
-		// Generate the UpdateStatus method if the type has a status
-		if genStatus(t) {
+	if !tags.NoVerbs && !tags.Readonly {
+		if !skipMethod("create", tags.SkipVerbs) {
+			sw.Do(createTemplate, m)
+		}
+		if !skipMethod("update", tags.SkipVerbs) {
+			sw.Do(updateTemplate, m)
+		}
+		if !skipMethod("updateStatus", tags.SkipVerbs) && genStatus(t) {
 			sw.Do(updateStatusTemplate, m)
 		}
-		sw.Do(deleteTemplate, m)
-		sw.Do(deleteCollectionTemplate, m)
+		if !skipMethod("delete", tags.SkipVerbs) {
+			sw.Do(deleteTemplate, m)
+		}
+		if !skipMethod("deleteCollection", tags.SkipVerbs) {
+			sw.Do(deleteCollectionTemplate, m)
+		}
 	}
 
-	if !noMethods {
-		sw.Do(getTemplate, m)
-		if hasObjectMeta(t) {
-			sw.Do(listUsingOptionsTemplate, m)
-		} else {
-			sw.Do(listTemplate, m)
+	if !tags.NoVerbs {
+		if !skipMethod("get", tags.SkipVerbs) {
+			sw.Do(getTemplate, m)
+		}
+		if !skipMethod("list", tags.SkipVerbs) {
+			if hasObjectMeta(t) {
+				sw.Do(listUsingOptionsTemplate, m)
+			} else {
+				sw.Do(listTemplate, m)
+			}
+		}
+		if !skipMethod("watch", tags.SkipVerbs) {
+			sw.Do(watchTemplate, m)
 		}
-		sw.Do(watchTemplate, m)
 	}
 
-	if !noMethods && !readonly {
-		sw.Do(patchTemplate, m)
+	if !tags.NoVerbs && !tags.Readonly {
```

Comment:
yeah, it makes sure the Patch() will be the last method (i can move it upper but that will reorder the methods... not sure if we care about that)